Zeeguu Reader/Zeeguu Reader/AddFeedTableViewController.swift

```diff
@@ -102,6 +113,7 @@ class AddFeedTableViewController: UITableViewController {
 	}
 	
 	func cancel(sender: UIBarButtonItem) {
+		urlField.resignFirstResponder()
```

Comment:
That line makes sure that the keyboard disappears the same time the addFeed view disappears. Without this line, the keyboard will disappear once the addFeed view disappeared completely, which is a bit late in my opinion.